locust/stats.py

```diff
@@ -822,13 +822,13 @@ def requests_csv(stats, csv_writer):
             s.name,
             s.num_requests,
             s.num_failures,
-            s.median_response_time,
-            s.avg_response_time,
-            s.min_response_time or 0,
-            s.max_response_time,
-            s.avg_content_length,
-            s.total_rps,
-            s.total_fail_per_sec,
+            round(s.median_response_time) if s.median_response_time else s.median_response_time,
```

Comment:
I think you can define a function above such as `_adjust_precision(stat)` instead of fixing the precision at this scope. Moreover, having a separate function allows customization using different precision, if needed.

Reply:
These changes are not intended to bring some new feature, but just to return an old view of csv report after regression.
Previously, data format was done also at this scope and was broken with an injection of csv_writer in 1.x.x version:
https://github.com/locustio/locust/pull/1428/files#diff-5d5f310549d6d596beaa43a1282ec49eL822
If we'll create a new function, it will be a complicated because of different data types in stats entries (time metrics, content metric, rate metrics). So it should be a some reason to add a new tangled function that probably will be used only once for this specific case.